R-packages/covidcast/R/covidcast.R

```diff
@@ -204,7 +211,7 @@ MAX_RESULTS <- 3649
 #'   [`msa_census`], [`state_census`]
 #' @export
 #' @importFrom rlang abort
-#' @importFrom dplyr %>%
+#' @importFrom dplyr %>% filter
```

Comment:
That was from the first draft, but was removed before I resubmitted.
More generally though, it ensures that the function would use `dplyr::filter` instead of another package's version